packages/frontend/core/src/components/explorer/docs-view/doc-list-item.tsx

```
@@ -0,0 +1,400 @@
+  static connector = '||';
+  static create(groupId: string, docId: string) {
+    return `${groupId}${this.connector}${docId}`;
+  }
+  static parse(mixId: string) {
+    if (!mixId) {
+      return { groupId: null, docId: null };
+    }
+    const [groupId, docId] = mixId.split(this.connector);
+    return { groupId, docId };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)